models/products-model.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+const ProductsSchema =  new Schema({
+  img: 
+      { data: Buffer, contentType: String }
+  , 
+  name : {
+    type : String , 
+    required : true
+  } , 
+  description : {
+    type : String ,
+    required : true
+  }
+    , 
+  price : {
+    type : Number ,  
+    required : true
+  } , 
+   
+  inStock : {
+    type : Boolean ,  
+    required : true
+  } , 
+  fastDelivery : {
+    type : Boolean ,  
+    required : true
+  } ,
+    quantity : {
+    type : Number ,  
+    required : true
+  } 
+}
+)
+
+const Products = mongoose.model('Products' , ProductsSchema , 'products')
```

Comment:
According to mongoose documentation, the first argument should always be the singular name of the collection for which the model is for. Mongoose automatically searches for the plural name for the same for our MongoDB database. 
So use `Product` instead of `Products`
Read more about it [here](https://mongoosejs.com/docs/models.html)